src/main/java/com/twilio/oai/TwilioCsharpGenerator.java

```diff
@@ -191,6 +191,12 @@ public OperationsMap postProcessOperationsWithModels(final OperationsMap objs, L
             resource.put("path", path);
             resource.put("resourceName", resourceName);
             resource.put("resourceConstant", "Resource");
+            if(enums.size() > 0){
+                resource.put("hasEnums", true);
+            }
+            if(arrayParamsPresent == true){
```

Comment:
if (arrayParamsPresent) is enough
Change it at other places as well. Try to have spaces between 
1. "if" and "("
 2. ")" and "{"